lib/Statocles/Store.pm

```diff
@@ -324,6 +324,24 @@ sub has_file {
     return $self->path->child( $path )->is_file;
 }
 
+=method files
+
+    my $iter = $store->files
+
+Returns an iterator which iterates over I<all> files in the store,
+regardless of type of file.  The iterator returns a L<Path::Tiny>
+object or undef if no files remain.  It is used by L<find_files>.
+
+=cut
+
+sub files {
+
```

Comment:
One minor formatting thing: Could you get rid of the two lines of whitespace in this sub?